src/components/ColorContainer.js

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+import ColorDiv from './ColorDiv';
+import ColorPicker from './ColorPicker';
+
+export default class ColorContainer extends PureComponent{
+  state={
+    color:'',
+    clicks:0
+  }
+ 
+  onClickHandler = (newColor)=>{
+    let clickCount = this.state.clicks;
+    clickCount++;
+    this.setState({
```

Comment:
when incrementing or adding to state and not replacing state, make sure to force react to bring in the previous state with something like : 
this.setState(state => {
      return { count: state.count + 1 };
    });